### DISCUSSION TITLE
Option to force Plain mode for email alerting

### DISCUSSION THREAD

**Discussion by bleizg** [jertel/elastalert2#593] [Q&A] [answered]
Hello,
I'm facing to an issue with the email alerter.

The email server I must use to emit email is supporting only the TLS1.0/1.1 protocol

Regarding the code [here ](https://github.com/jertel/elastalert2/blob/9bf8a05b03d1ec86f66da63dd0b39bb31e7d5972/elastalert/alerters/email.py#L107-L108), Elastalert detects the TLS capabilities of the email server and then use this protocol.

As Debian Buster does not support anymore old version of TLS, here is the exception generated by Elastalert:
`ERROR:elastalert:Error while running alert email: Error connecting to SMTP host: EOF occurred in violation of protocol (_ssl.c:997)`

What do you think about having an option to force the PLAIN mode fpr the Email Alerter?

Thanks :)

**Comment by jertel** [comment, marked as answer]
The Elasticsearch and other connections from ElastAlert allow both secure and insecure modes, so perhaps it could exist as an option, provided it doesn't allow insecure connections by default.
